BaseApp/COD57000.TXT

```diff
@@ -0,0 +1,40 @@
+OBJECT Codeunit 57000 DotNet Add Text Encoding
+{
+  OBJECT-PROPERTIES
+  {
+    Date=18-06-14;
```

Comment:
Please clear date, time and remove modified flag.

Reply:
cleared and modified.